oprf/oprf_test.go

```diff
@@ -12,6 +12,46 @@ import (
 	"github.com/cloudflare/circl/oprf/group"
 )
 
+func TestServerSerialization(t *testing.T) {
+	suite, err := suiteFromID(OPRFP256, []byte(""))
+	if err != nil {
+		t.Fatal(err)
+	}
+	keyPair := GenerateKeyPair(suite)
+
+	server, err := NewServerWithKeyPair(OPRFP256, keyPair)
+	if err != nil {
+		t.Fatal("invalid setup of server: " + err.Error())
+	}
+
+	input := []byte("hello world")
+	info := []byte("info")
+	outputA, errA := server.FullEvaluate(input, info)
```

Comment:
Why is this using a new error variable and not reusing the previous one?

Reply:
I tend to think it's more readable code this way.